angular_token_auth.js

```diff
@@ -13,6 +13,8 @@
         AUTH_HEADER_PREFIX: 'Token',
         ALLOWED_HOSTS: [],
         COOKIE_PATH: null
+        // Optional settings examples:
+        // STORAGE_METHOD: 'localstorage',
```

Comment:
So:
* Add a default option with a value that does nothing
OR
* Have no default option but give an example with a real value that is not the default